includes/cf/sonarr-streaming-services.md

```diff
@@ -1,6 +1,6 @@
 ??? abstract "Streaming Services - [CLICK TO EXPAND]"
     | Custom Format                                                                           | Score                                     | Trash ID                               |
-    | --------------------------------------------------------------------------------------- | ----------------------------------------- | -------------------------------------- |
+    | :-------------------------------------------------------------------------------------- | :---------------------------------------- | :------------------------------------- |
```

Comment:
What does the `:` do at the top of a table?

Reply:
It's for alignment
"Alignment
You can align text in the columns to the left, right, or center by adding a colon (:) to the left, right, or on both side of the hyphens within the header row."

Not specifically needed to vscode addon that aligns tables chucked it in. I'll just remove it

Reply:
Thanks! TIL.